rules/rules-reviewed/eap8/eap7/picketlink.windup.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="picketlink"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis of Java EE applications that need to remove their picketlink configuration
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final"/>
+        </dependencies>
+        <targetTechnology id="eap" versionRange="[8,9)"/>
+    </metadata>
+    <rules>
+        <rule id="picketlink-00010">
+            <when>
+                <or>
+                    <file filename="{*}WEB-INF/{*}picketlink.xml"/>
+                    <filecontent pattern="picketlink" filename="{*}.xml"/>
+                    <file filename="picketlink-sts.xml"/>
+                    <filecontent pattern="PicketLinkSTS" filename="picketlink.xml"/>
```

Comment:
I have noticed the current rule does not contain links for the user to understand better the issue.
From the document attached to the ticket I get this info:

_If a picketlink-sts.xml file or occurrences of “PicketLinkSTS” were detected in the picketlink.xml file, the user will need to configure Apache CXF STS instead as described briefly [here](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.4/html-single/developing_web_services_applications/index#ws_trust_sts_apache_CXF_support). (If more details are needed here, we’ll need to ask the Web Services team to provide this information.)_

Is there a reason for not including the link and described above?